### PR TITLE
Make some corrections

### PR DESCRIPTION
**Pull request by chenrui333**
Some minor changes.

### CODE REVIEW

**Comment by chenrui333** [comment]
updated the `build` goal
```
$ make install
go get -d -t -v ./...
go get github.com/golang/lint/golint
go build -o terraform-provider-launchdarkly
install -d -m 755 ~/.terraform.d/plugins
install terraform-provider-launchdarkly ~/.terraform.d/plugins
```

**Review by mlafeldt — COMMENTED** [review]
Thanks for the contribution. See my comments.

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ resource launchdarkly_feature_flag "example" {
 
   tags = [
     "hello",
-    "world",
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply README.md]: ah, did not know that. I remove this line change.

File: Makefile [hunk 2]
```diff
@@ -28,5 +28,5 @@ apply: build
 	terraform $@
 
 destroy: build
-	terraform init
+	terraform destroy
 	terraform $@
```
[the remark this answers is not in the document]
Comment (chenrui333) [reply Makefile]: So `$@` would expand into `terraform destroy`?

**Comment by chenrui333** [comment]
@mlafeldt another question I have is the PR build seems not get built per circleci?

**Comment by chenrui333** [comment]
I reverted some of the changes, please give another look, thanks!

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (mlafeldt) [reply Makefile]: Here, `$@` expands to the name of the target (`destroy`), see https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html

[the remark this answers is not in the document]
Comment (chenrui333) [reply Makefile]: I see, thanks for sharing.

**Comment by mlafeldt** [comment]
> @mlafeldt another question I have is the PR build seems not get built per circleci?

Good catch. I enabled building of forked pull requested in CircleCI.

**Pull request merged**